cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ExecutorState.java

```diff
@@ -15,7 +15,7 @@
 import static com.linkedin.kafka.cruisecontrol.executor.ExecutionTaskTracker.ExecutionTasksSummary;
 
 public class ExecutorState {
-  private static final String TRIGGERED_USER_TASK_ID = "triggeredUserTaskId";
+  private static final String TRIGGERED_TASK_ID = "triggeredTaskId";
```

Comment:
Should we refrain from changing the existing field name to avoid introducing a backwards incompatible change (e.g. this change will potentially break the `CCUI`)? Can we instead introduce a new field, such as `triggeredSelfHealingTaskId` to show task ids triggered by self-healing?